cg/server/endpoints/orders.py

```diff
@@ -175,6 +175,9 @@ def submit_order(order_type: OrderType):
                 content=request_json, file_format=FileFormat.JSON
             ),
         )
+        request.json["delivery_type"] = "fastq"
```

Comment:
This should be solved in the FE in https://github.com/Clinical-Genomics/clinical-genomics-ui/issues/606